src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -581,13 +581,19 @@ public AddressBalancesResult GetAddressBalances(string[] addresses, int minConfi
         /// <inheritdoc />
         public VerboseAddressBalancesResult GetVerboseAddressBalancesData(string[] addresses)
         {
+            var result = new VerboseAddressBalancesResult()
```

Comment:
You don't need these changes as if you are always passing in empty list of addresses (if its null on the otherside). The `foreach` takes care of this.

Reply:
This bypasses `isQueryable` due to the address index not being required.